packages/core/src/dialects/abstract/query-interface.types.ts

```diff
@@ -134,3 +136,31 @@ export interface RemoveConstraintOptions extends RemoveConstraintQueryOptions, Q
 
 /** Options accepted by {@link AbstractQueryInterface#showConstraints} */
 export interface ShowConstraintsOptions extends ShowConstraintsQueryOptions, QueryRawOptions { }
+
+/** Options accepted by {@link AbstractQueryInterface#createSavepoint} */
+export interface CreateSavepointOptions extends QueryRawOptions {
+  savepointName: string;
+}
+
+/** Options accepted by {@link AbstractQueryInterface#startTransaction} */
+export interface StartTransactionOptions extends StartTransactionQueryOptions, QueryRawOptions { }
+
+/** Options accepted by {@link AbstractQueryInterface#commitTransaction} */
+export interface CommitTransactionOptions extends QueryRawOptions {
+  transactionName: string;
```

Comment:
this doesn't seem to be used, same in RollbackTransactionOptions

Reply:
Removed now